src/lpmd_proc.c

```diff
@@ -179,6 +189,7 @@ static int lpm_can_process(enum lpm_command cmd)
 		case USER_AUTO:
 			lpm_state &= ~LPM_USER_ON;
 			lpm_state &= ~LPM_USER_OFF;
+			lpm_state |= LPM_UTIL_ON;
```

Comment:
This means we always want utilization monitor in AUTO mode.
Previously, if we use HFI hints only, then we never pull. So this is a functional change.